configs/luslab-base.config

```diff
@@ -58,6 +58,9 @@ process.shell = ['/bin/bash', '-euo', 'pipefail']
 //once this is established and works well, nextflow might implement this behavior as new default.
 docker.runOptions = '-u \$(id -u):\$(id -g)'
 
+// Path to shared luslab singularity cache
+singularity.cacheDir = '/camp/lab/luscomben/home/shared/singularity'
```

Comment:
What if we run this locally? shouldnt this be in the crick config?

Reply:
Agree and moved